nomad/structs/csi_test.go

```diff
@@ -42,7 +42,7 @@ func TestCSIVolumeClaim(t *testing.T) {
 	require.NoError(vol.Claim(claim, alloc1))
 	require.True(vol.ReadSchedulable())
 	require.False(vol.WriteSchedulable())
-	require.False(vol.WriteFreeClaims())
+	require.False(vol.HasFreeWriteClaims())
```

Comment:
Semgrep doesn't like me touching this without updating all the `require` calls in the entire test. Should we just go ahead and do that here, @lgfa29 ?

Reply:
Tell you what, given this is a refactoring PR anyways, I might as well do it now and reduce semgrep noise in the future.